src/core/utils.ts

```diff
@@ -113,7 +113,13 @@ export function reverseDeepMerge(dest, source = undefined) {
         if (isFunction(src[srcArg])) {
           dest[srcArg] = src[srcArg];
         } else {
-          dest[srcArg] = clone(src[srcArg]);
+          try {
```

Comment:
I tried to find a sane/simple way of checking if the property was readonly but found none, that is why I decided for the try catch.